docs/UserGuide.md

```diff
@@ -662,12 +679,24 @@ worries! You can edit the details of a customer simply by calling this command.
 
 **Format:** `customer edit CUSTOMER_ID [--name NAME] [--phone PHONE_NUMBER] [--email EMAIL] [--address ADDRESS]`
 
-<box background-color="#dff0d8" border-color="#d6e9c6" type="info" icon=":information_source:">
-    **Note:** [ ] around a parameter indicates that it is optional.
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+
+* At least one of the optional fields must be provided.
+* `CUSTOMER_ID`{.swfit} must be an integer greater than 0.
```

Comment:
`CUSTOMER_ID`{.swift} must be an integer greater than 0 that corresponds to an existing Customer in the address book.`